src/components/Rum/Rum.js

```diff
@@ -13,7 +13,7 @@ const Rum = () => {
     getRumRecipes()
     .then(data => data.drinks)
```

Comment:
Why not combine these into one line with:
`.then(data => setRumContainer(data.drinks))`